quartz/plugins/transformers/ofm.ts

```diff
@@ -380,9 +380,13 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                 const defaultState = collapseChar === "-" ? "collapsed" : "expanded"
                 const titleContent =
                   match.input.slice(calloutDirective.length).trim() || capitalize(calloutType)
+                const hasTitle =
+                  restChildren.length > 0 ? restChildren[0].children[0]?.type === "text" : false
```

Comment:
Hmm I have bashed my head and couldn't figure out the type of this correctly, but the object apparently has children here.